projects/1.20/assets/meet-your-fight/meetyourfight/lang/zh_cn.json

```
@@ -0,0 +1,149 @@
+	"item.meetyourfight.spectres_eye": "幽灵之眼",
+	"item.meetyourfight.spectres_eye.desc": "使半径%1$d格内的敌对生物发光",
+	"item.meetyourfight.spectres_grasp": "幽灵之握",
+	"item.meetyourfight.phantasmal_rifle": "幻影步枪",
+	"item.meetyourfight.phantasmal_rifle.desc": "弹射物能够穿过方块",
+	"item.meetyourfight.aether_glazed_cupcake": "浇汁以太蛋糕",
+
+	"item.meetyourfight.devils_ante": "恶魔赌注",
+	"item.meetyourfight.devils_ante.desc": "召唤福尔图娜女爵",
+	"item.meetyourfight.devils_ante.gear": "钻石质",
+	"item.meetyourfight.fortunes_favor": "命运之眷",
+	"item.meetyourfight.slicers_dice": "切割者之骰",
+	"item.meetyourfight.slicers_dice.desc": "攻击时有%1$d%%概率额外造成%2$d%%的伤害",
+	"item.meetyourfight.ace_of_iron": "铁之王牌",
+	"item.meetyourfight.ace_of_iron.desc": "有%1$d%%概率无视受到的攻击",
+	"item.meetyourfight.cocktail_cutlass": "甜心鸡尾酒之刃",
+	"item.meetyourfight.cocktail_cutlass.desc": "攻击时有%1$d%%概率给予目标药水效果",
+	"item.meetyourfight.cocktail_shotgun": "猎人霰弹枪",
+	"item.meetyourfight.cocktail_shotgun.desc": "有%1$d%%概率射出带负面效果的喷溅型药水",
+	"item.meetyourfight.velvet_fortune": "幸运天鹅绒",
+
+	"item.meetyourfight.desc.luck": "会被幸运影响",
+
+	"item.meetyourfight.fossil_bait": "化石之饵",
+	"item.meetyourfight.fossil_bait.desc": "召唤沼泽巨颌怪",
+	"item.meetyourfight.fossil_bait.gear": "石质或铁质",
+	"item.meetyourfight.mossy_tooth": "藓牙",
+	"item.meetyourfight.bone_raker": "骨质钉耙",
+	"item.meetyourfight.depth_star": "深境之星",
+	"item.meetyourfight.depth_star.desc": "可以释放短程冲击波",
+	"item.meetyourfight.depth_star.desc2": "其伤害随普通攻击伤害的增加而增加",
+	"item.meetyourfight.caged_heart": "囚笼之心",
+	"item.meetyourfight.caged_heart.desc": "降低%2$d%%超过最大生命值%1$d%%的伤害",
+	"item.meetyourfight.marshy_delight": "湿软泥泞的佳肴",
+
+	"item.meetyourfight.dusk_key": "黄昏之钥",
+	"item.meetyourfight.dusk_key.desc": "召唤薄暮之锋，罗莎琳",
+	"item.meetyourfight.dusk_key.gear": "下界合金质",
+	"item.meetyourfight.violet_bloom": "暮光紫罗兰",
+	"item.meetyourfight.twilights_thorn": "胧暮荆棘",
+	"item.meetyourfight.wilted_ideals": "未竟之志",
+	"item.meetyourfight.wilted_ideals.desc": "增加%1$d%%所有种类的伤害",
+	"item.meetyourfight.blossoming_mind": "绽放之念",
+	"item.meetyourfight.blossoming_mind.desc": "增加%1$d%%生物掉落的经验量",
+	"item.meetyourfight.tomb_planter": "垂死之辉",
+	"item.meetyourfight.tomb_planter.desc": "催熟你杀死生物之处附近的植物",
+	"item.meetyourfight.petal_cream": "花瓣冰淇淋",
+
+	"item.meetyourfight.music_disc_swampjaw": "音乐唱片",
+	"item.meetyourfight.music_disc_swampjaw.desc": "Kain Vinosec - Shadows Falling In Lush Forests",
+	"item.meetyourfight.music_disc_bellringer": "音乐唱片",
+	"item.meetyourfight.music_disc_bellringer.desc": "Monkey Warhol - Magnum",
+	"item.meetyourfight.music_disc_fortuna": "音乐唱片",
+	"item.meetyourfight.music_disc_fortuna.desc": "PinpinNeon - Place Your Bet ~ Life On Red",
+	"item.meetyourfight.music_disc_rosalyne": "音乐唱片",
+	"item.meetyourfight.music_disc_rosalyne.desc": "Guifrog - Frog Punch",
+
+	"block.meetyourfight.bellringer_head": "敲钟人的头",
+	"block.meetyourfight.dame_fortuna_head": "福尔图娜女爵的头",
+	"block.meetyourfight.swampjaw_head": "沼泽巨颌怪的头",

```

```suggestion
	"block.meetyourfight.swampjaw_head": "沼泽巨颌怪微缩头颅",
```
Skull需要译作术语“头颅”，Downscaled漏译

---

> Skull需要译作术语“头颅”，Downscaled漏译

嗯确实这玩意是个骨头，所以头颅没问题